cmd/kubeadm/app/phases/certs/pkiutil/pki_helpers.go

```diff
@@ -123,23 +180,22 @@ func TryLoadCertAndKeyFromDisk(pkiPath, name string) (*x509.Certificate, *rsa.Pr
 	case *ecdsa.PrivateKey:
```

Comment:
I think we decided to only support RSA, so the ECDSA TODO and check don't make sense anymore